met-web/src/components/engagement/dashboard/report/EngagementDashboard.tsx

```diff
@@ -33,6 +33,16 @@ export const EngagementDashboard = () => {
             throw new Error('Engagement has not yet been opened');
         }
     };
+
+    const validateEngagement = (engagementToValidate: Engagement) => {
+        const isClosed = engagementToValidate?.submission_status === SubmissionStatus.Closed;
+        const isPublicUser = !roles.includes('access_dashboard');
```

Comment:
plz change the variable name is canAccessDashboard

Reply:
changed it , thanks